components/custom-button.tsx

```diff
@@ -0,0 +1,13 @@
+import { blueDarkAlt } from '@pxblue/react-native-themes';
+import React from 'react';
+import { Button, useTheme } from 'react-native-paper';
+
+export const MyCustomButton: typeof Button = (props) => {
+    const theme = useTheme(props.theme);
+    return (
+        <Button
+            {...props}
+            theme={props.mode === 'contained' && theme.dark ? Object.assign({}, blueDarkAlt, props.theme) : {}}
```

Comment:
why do we not have to do cloneDeep on this one but we do in the input? Should we be doing it here?

Reply:
We don't need it here. The TextInput component is the only one that needs it. I made false assumptions about this component but the bug was just a result of the TextInput issue.